### ISSUE TITLE
Static registers for functions

### ISSUE THREAD

**Issue by marekjm** [marekjm/wudoovm#22]
Implement static registers for functions.

**Comment by marekjm** [comment]
Think about how to let programmers tell how many static registers they want for given function. Curretnly, 16 static registers are assigned (by default) for each function that requests a static register set.

Maybe sytax of assembler function signatures should be altered? Suggestion:

```
.def: <name> <returns-or-not>, <local-registers>, <static-registers>
  <instruction>+
.end
```

**Comment by marekjm** [comment]
Static registers are merged into devel branch in commit 4f77dd5.
What is left is creating custom amount of them.

**Comment by marekjm** [comment]
Adjusting register set sizes is the topic of isse #36 .
